backend/data_handler_app/models.py

```diff
@@ -22,41 +22,29 @@ class Answer(models.Model):
     def __str__(self):
         return self.answer
 
-class ClientData(models.Model):
```

Comment:
We need to make sure to remove reference/import of `ClientData` in the codebase since it'll no longer exist